editor/inserter/menu.js

```diff
@@ -33,6 +33,7 @@ class InserterMenu extends Component {
 		this.onKeyDown = this.onKeyDown.bind( this );
 		this.getVisibleBlocks = this.getVisibleBlocks.bind( this );
 		this.sortBlocksByCategory = this.sortBlocksByCategory.bind( this );
+		this.debouncedSpeak = debounce( this.speakAssertive.bind( this ), 500 );
```

Comment:
Also, if the amount of time will be used in other components, I wonder if it may be better to make it a constant.